diff --git a/mm-video-v4l2/vidc/vdec/Android.mk b/mm-video-v4l2/vidc/vdec/Android.mk
old mode 100644
new mode 100755
index 439be53..8eaf9be
--- a/mm-video-v4l2/vidc/vdec/Android.mk
+++ b/mm-video-v4l2/vidc/vdec/Android.mk
@@ -114,16 +114,20 @@ libmm-vdec-inc          += $(OMX_VIDEO_PATH)/vidc/common/inc
 libmm-vdec-inc          += hardware/qcom/media-caf-new/mm-core/inc
 #DRM include - Interface which loads the DRM library
 libmm-vdec-inc	        += $(OMX_VIDEO_PATH)/DivxDrmDecrypt/inc
-libmm-vdec-inc          += hardware/qcom/display-caf-new/libgralloc
+#libmm-vdec-inc          += hardware/qcom/display-caf-new/libgralloc
+libmm-vdec-inc          += hardware/qcom/display-caf/libgralloc
 libmm-vdec-inc          += frameworks/native/include/media/openmax
 libmm-vdec-inc          += frameworks/native/include/media/hardware
 libmm-vdec-inc          += $(vdec-inc)
-libmm-vdec-inc          += hardware/qcom/display-caf-new/libqdutils
+#libmm-vdec-inc          += hardware/qcom/display-caf-new/libqdutils
+libmm-vdec-inc          += hardware/qcom/display-caf/libqdutils
 libmm-vdec-inc      += hardware/qcom/media-caf-new/libc2dcolorconvert
-libmm-vdec-inc      += hardware/qcom/display-caf-new/libcopybit
+libmm-vdec-inc      += hardware/qcom/display-caf/libcopybit
+#libmm-vdec-inc      += hardware/qcom/display-caf-new/libcopybit
 libmm-vdec-inc      += frameworks/av/include/media/stagefright
 libmm-vdec-inc      += $(TARGET_OUT_HEADERS)/mm-video/SwVdec
-libmm-vdec-inc      += $(TARGET_OUT_HEADERS)/qcom/display-caf-new/
+libmm-vdec-inc      += $(TARGET_OUT_HEADERS)/qcom/display-caf/
+#libmm-vdec-inc      += $(TARGET_OUT_HEADERS)/qcom/display-caf-new/
 
 ifneq ($(call is-platform-sdk-version-at-least, 19),true)
 libOmxVdec-def += -DMETADATA_FOR_DYNAMIC_MODE
@@ -155,7 +159,8 @@ ifeq ($(call is-board-platform-in-list,msm8974 msm8610 msm8226 apq8084 mpq8092 m
 LOCAL_SRC_FILES         += src/omx_vdec_msm8974.cpp
 else
 LOCAL_SHARED_LIBRARIES  += libhardware
-libmm-vdec-inc          += hardware/qcom/display-caf-new/libhwcomposer
+libmm-vdec-inc          += hardware/qcom/display-caf/libhwcomposer
+#libmm-vdec-inc          += hardware/qcom/display-caf-new/libhwcomposer
 LOCAL_SRC_FILES         += src/power_module.cpp
 LOCAL_SRC_FILES         += src/omx_vdec.cpp
 endif
diff --git a/mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp b/mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp
old mode 100644
new mode 100755
